src/validation-controller.ts

```diff
@@ -47,14 +48,20 @@ export class ValidationController {
   /**
    * The trigger that will invoke automatic validation of a property used in a binding.
    */
-  public validateTrigger = validateTrigger.blur;
+  public validateTrigger: validateTrigger;
 
   // Promise that resolves when validation has completed.
   private finishValidating: Promise<any> = Promise.resolve();
 
   private eventCallbacks: ((event: ValidateEvent) => void)[] = [];
 
-  constructor(private validator: Validator, private propertyParser: PropertyAccessorParser) { }
+  constructor(
+    private validator: Validator,
+    private propertyParser: PropertyAccessorParser,
+    config: AureliaValidationConfiguration,

```

Comment:
I dont think we can afford to assume that this is always supplied. Folks might have just done `new ValidationController(new Validator, new parser)` and this will break. We can do:
```ts
  constructor(
    private validator: Validator,
    private propertyParser: PropertyAccessorParser,
    config?: AureliaValidationConfiguration,
  ) {
    this.validateTrigger = config instanceof AureliaValidationConfiguration ? config.getDefaultValidationTrigger() : validateTrigger.blur;
  }
```

Reply:
I'll put `validateTrigger.blur` in a static property of `AureliaValidationConfiguration` to prevent duplication. Otherwise no problem.